lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
```

Comment:
This will add a dependency on the presence of a .fog file for executing this code.  I think that we could reasonable add that (and update documentation to reflect it), or we need to add a switch to beaker to attempt loading in the case of 'vpool_auth = true' or some such.